README.md

```diff
@@ -56,6 +56,21 @@ Other pieces we hope to achieve in 2018:
 And some projects are private for the moment, but we hope to make them public
 as soon as possible.
 
+### Release v3.1.0
+
+The release introduces the notion of signature scheme for a given skipchain so that
+one can define which co-signing algorithm will be used to sign the forward links. This
+was necessary in the context of weaknesses in the BLS signature algorithm (see the
+[paper](https://crypto.stanford.edu/~dabo/pubs/papers/BLSmultisig.html)). New
+skipchains will be created with [BDN](https://github.com/dedis/kyber/tree/master/sign/bdn)
+set as the signature scheme.
+
+Because of a new scheme is default, that means that skipchains created after v3.1.0
```

Comment:
is there an option to create pre-3.1.0 skipchains when running 3.1+?

Reply:
Not without a small change to use the signature scheme provided when creating a genesis block.

Reply:
i guess my question is more like: should we allow such an option? what are the pros/cons?

Reply:
@jeffallen ?

Reply:
Jeff confirmed that we shouldn't allow it.